### ISSUE TITLE
Cannot read property end of undefined

### ISSUE THREAD

**Issue by SebastianStehle** [matthias-schuetz/karma-htmlfile-reporter#18]
Hi,

I am running the reporter with webpack. Whenever my test fails, I get the error above, which is a little bit annoying:

It is at 145:44

**Comment by matthias-schuetz** [comment]
Did you update to 0.3.2?
I can't currently produce any errors with the newest version.

**Comment by SebastianStehle** [comment]
Yes, still happens

![untitled](https://cloud.githubusercontent.com/assets/1236435/16173545/2115e36a-35a4-11e6-8859-f08d7cff66d3.png)

**Comment by matthias-schuetz** [comment]
OK, thanks for the additional info! I could finally reproduce an error like that.
The error handling of the reporter wasn't final yet so I completed that. I also added error handling for errors that break the test progress (like in your case): here the HTML file will be generated but contain the error that broke the tests. So you won't see any results (since there aren't any) but at least the message that comes from Karma. Now, there should be no case where the reporter will crash. It's already on npm with 0.3.3.
